### ISSUE TITLE
Import the CSS from gh-pages

### ISSUE THREAD

**Issue by arthursoares** [envisioning/tdb#161]
There are some fixes, like cursor type for dashboard and color.

**Comment by arthursoares** [comment]
Please @rafaelcorreiapoli @lnmunhoz let's make a process for this to work seamlessly, it would be a lot easier for us to use the styles that I'm working on. There are loads of small differences in the two versions of the site (tdb-staging and tdb-design). Is this happening because you're rewritting the classes? Check #166 for instance.

What is the best way to achieve this? Create a separate repo for the style folder and sync it within our main repo? Help with this.

**Comment by rafaelcorreiapoli** [comment]
@arthursoares, I discussed this a lot with @lnmunhoz yesterday. He will try something with Gulp (and will explain to you today). But, im *my* opnion, it's working as it is now. mainDashboard was an isolated problem and we fixed it, I'm sorry for that.
In #150 for instance, I imported all the styles correctly and paid attention to every single detail. In the beginning I was importing the CSS from a wrong file and was having some problems, but then you told me the correct file and I fixed it

**Comment by lnmunhoz** [comment]
I'm creating the integration right now

**Comment by arthursoares** [comment]
@rafaelcorreiapoli oh, OK. If it is working for you, then it is fine for me :) Whatever works for both.

**Comment by lnmunhoz** [comment]
I just created a new build system for front-end development that we can host on gh-pages and shares the same style files. I will discuss about it with @rafaelcorreiapoli and then create a guide for you @arthursoares. Its based on handlebars and less.

**Comment by rafaelcorreiapoli** [comment]
What I propose:
our main branch will have two separate folders:
```app```
and
```styleguide```

inside ```app```, we will put our real project, the one we are working on (with Meteor build system, which is slow for designing purposes)

inside ```styleguide``` we will put a simple node.js server with a build system that uses METEOR's (from the folder ```.../app/.../stylesheets``` as sources to build css.
This server will have a light weight build system (not the heavy one from meteor that is causing problems) and the designers will be able to work with normal html, with a fast build. All the styles that people working on the ```styleguide``` project should be placed on the REAL PROJECT (```app```)
To sum up, we will have two projects (heavy weight build and light weight build) that will consume the same Less files to build.

**Comment by arthursoares** [comment]
@rafaelcorreiapoli I'll need some help with this flow. I don't know how to create and correlate pages in Meteor. What I'm doing right now is creating tiny components / elements in PHP and loading them as requested in each page. Like this: https://github.com/envisioning/tdb/tree/gh-pages/elements

**Comment by rafaelcorreiapoli** [comment]
We will guide you, don't worry. And this will be experimental, if you don't like it we can keep things how they are now..but we are hoping that this improvement will lead us to higher productivity

**Comment by lnmunhoz** [comment]
@arthursoares Take a look at the [documentation](https://github.com/envisioning/tdb/tree/development/styleguide) of the styleguide project. The project doesn't have all your work from `gh-pages`.

**Comment by arthursoares** [comment]
This is awesome guys :)
